blueprints/ember-bootstrap/index.js

```diff
@@ -186,14 +186,21 @@ module.exports = {
 
     settings.importBootstrapCSS = (preprocessor === 'none');
 
-    if (fs.existsSync(file)) {
-      this.ui.writeLine(chalk.green(`Added ember-bootstrap configuration to ${file}`));
-      let source = fs.readFileSync(file, 'utf-8');
-      let build = new BuildConfigEditor(source);
-      let newBuild = build.edit('ember-bootstrap', settings);
-      fs.writeFileSync(file, newBuild.code());
-    } else {
+    if (!fs.existsSync(file)) {
       this.ui.writeLine(chalk.red(`Could not find ${file} to modify.`));
+      return;
+    }
+
+    this.ui.writeLine(chalk.green(`Added ember-bootstrap configuration to ${file}`));
```

Comment:
Done. All the others are optimistic, but this makes sense to be more pessimistic under the circumstances.